pkg/capabilities/v2/protoc/pkg/test_capabilities/importclash/clash.pb.go

```diff
@@ -0,0 +1,69 @@
+// Code generated by protoc-gen-go. DO NOT EDIT.
+// versions:
+// 	protoc-gen-go v1.36.6
+// 	protoc        v5.29.3
+// source: capabilities/v2/protoc/pkg/test_capabilities/importclash/clash.proto
+
+package importclash
+
+import (
+	_ "github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/protoc/pkg/pb"
+	pb "github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/protoc/pkg/test_capabilities/importclash/p1/pb"
+	pb1 "github.com/smartcontractkit/chainlink-common/pkg/capabilities/v2/protoc/pkg/test_capabilities/importclash/p2/pb"
```

Comment:
why do these get import as `pb` and `pb1` here but `pb3` and `pb4` in `clash_sdk_gen.go`

Reply:
The first file attempts the same import multiple times and dedups, the latest number is used. I can change to be consistant, this was a less invasive change

Reply:
ok, I see since it's generated I think it's fine

Reply:
This is proving more annoying to change than I thought it would be, the imports is for everything compiled include the files themselves and it's hard to difference what's what. Ok if I leave it?